src/index.ts

```diff
@@ -15,9 +15,9 @@ async function main () {
   }
   const goModDir = path.dirname(goModPath)
 
-  let goBuildTarget = path.normalize(core.getInput('go-build-target'))
```

Comment:
are we sure it's a good idea to _never_ normalize the path? I would think we could still do it as long as we have a special case for `./...`

Reply:
I'm not sure, but it seems like an aesthetic concern to me and I'm not sure what bad would come of it.

Reply:
fair enough